### PR TITLE
Add logic and tests

### PR DESCRIPTION
**Pull request by neggertz**

### CODE REVIEW

File: Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM elixir:1.7.3
```
Comment (spencerdcarlson) [inline Dockerfile.test]: Do we need a dockerfile if this is a lib?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Brief context on code (tell us why these changes are necessary)
```
Comment (spencerdcarlson) [inline PULL_REQUEST_TEMPLATE.md]: Cool!

File: lib/http_client.ex [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  @type error_message :: String.t()
+
+  @conversations_path "/api/v2/conversations"
+  @switchboard_invitations_path "/api/v2/switchboard_invitations"
+
+  def host, do: Application.get_env(:ex_kumete, :kumete_host)
+  def scheme, do: Application.get_env(:ex_kumete, :kumete_scheme)
+  def port, do: String.to_integer(Application.get_env(:ex_kumete, :kumete_port))
+
+  def create_switchboard_invitation(token, params) do
```
Comment (spencerdcarlson) [inline lib/http_client.ex]: specs?

File: lib/http_client.ex [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          required(String.t()) => String.t()
+        }
+  @type status_code :: integer
+  @type response_message :: String.t()
+  @type error_message :: String.t()
+
+  @conversations_path "/api/v2/conversations"
+  @switchboard_invitations_path "/api/v2/switchboard_invitations"
+
+  def host, do: Application.get_env(:ex_kumete, :kumete_host)
```
Comment (spencerdcarlson) [inline lib/http_client.ex]: defp?

File: lib/http_client.ex [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  def scheme, do: Application.get_env(:ex_kumete, :kumete_scheme)
+  def port, do: String.to_integer(Application.get_env(:ex_kumete, :kumete_port))
+
+  def create_switchboard_invitation(token, params) do
+    Logger.info("[KumeteClient] Creating switchboard invitation [#{inspect(params)}]")
```
Comment (spencerdcarlson) [inline lib/http_client.ex]: I just noticed this in Patronus, but I think we can configure the logger's metadata to pull the module name at runtime so we don't have to add it manually,

```elixir
config :logger,
  console: [
    format: "$time [$level] [$metadata] $message\n",
    metadata: [:module]
  ],
  level: :info,
  discard_threshold: 10000,
  sync_threshold: 250
```

File: Dockerfile.test (hunk 1 above)
Comment (neggertz) [reply Dockerfile.test]: Yes for codeship builds.

File: lib/http_client.ex (hunk 3 above)
Comment (neggertz) [reply lib/http_client.ex]: Added

**Pull request merged**